verify_files.py

```diff
@@ -8,22 +9,22 @@
     print "Unable to load file: set_info\nException information:\n" + str(ex.args)
     sys.exit("Unable to load file: set_info")
 try:
-    with open('cards_manual') as data_file:
-        manual_sets = commentjson.load(data_file)
+    with open('cards_manual.yml') as data_file:
+        manual_sets = yaml.load(data_file)
```

Comment:
`yaml.safe_load` should be better!
http://pyyaml.org/wiki/PyYAMLDocumentation#LoadingYAML